src/whylogs/whylabs_client/wrapper.py

```diff
@@ -89,7 +89,11 @@ def _upload_whylabs(dataset_profile, dataset_timestamp, profile_path):
         raise EnvironmentError('Missing dataset ID. Specify it via "datasetId" tag or WHYLABS_DEFAULT_DATASET_ID environment variable')
     try:
         with open(profile_path, "rb") as f:
-            log_api.log(org_id=org_id, model_id=dataset_id, dataset_timestamp=dataset_timestamp, file=f)
+            result = log_api.log_async(org_id=org_id, dataset_id=dataset_id) // dataset_timestamp=dataset_timestamp) looks like timestamp is built in whylabs client ?
+            http_response = requests.put(result["upload_url"], data=f.read())
+            assert (
```

Comment:
Rather than assert when the request returns something other than 'ok' (e.g. 200), let's log as error, we don't need to raise in this case we just need to communicate the status, its also inside a try block already and non 200 returns are valid failure cases modeled by the result already.